gsa/src/web/pages/reports/triggeralertdialog.js

```diff
@@ -56,7 +57,9 @@ const TriggerAlertDialog = ({
   filter = {},
   includeNotes = COMPOSER_CONTENT_DEFAULTS.includeNotes,
   includeOverrides = COMPOSER_CONTENT_DEFAULTS.includeOverrides,
+  showThresholdMessage,
```

Comment:
Same here. Add the default or require the prop.
```suggestion
  showThresholdMessage = false,
```